pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -322,6 +326,10 @@ func (r *ScaleREST) Update(ctx context.Context, name string, objInfo rest.Update
 	return newScale, false, nil
 }
 
+func (r *ScaleREST) ConvertToTable(ctx context.Context, object runtime.Object, tableOptions runtime.Object) (*metav1.Table, error) {
+	return r.store.ConvertToTable(ctx, object, tableOptions)
```

Comment:
Will this give the deployment's table columns, or scale-specific table columns?

Reply:
Scale specific table columns.